docs/source/api/v4/roles.rst

```diff
@@ -83,19 +80,19 @@ Response Structure
 	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
 	Whole-Content-Sha512: TEDXlQqWMSnJbL10JtFdbw0nqciNpjc4bd6m7iAB8aymakWeF+ghs1k5LayjdzHcjeDE8UNF/HXSxOFvoLFEuA==
 	X-Server-Name: traffic_ops_golang/
-	Date: Wed, 04 Sep 2019 17:15:36 GMT
-	Content-Length: 120
+	Date: Wed, 25 Aug 2021 20:10:34 GMT
+	Content-Length: 1608
 
 	{ "response": [
 		{
-			"id": 4,
 			"name": "admin",
-			"description": "super-user",
-			"privLevel": 30,
-			"capabilities": [
+			"description": "Has access to everything.",
+			"permissions": [
 				"all-write",
-				"all-read"
-			]
+				"all-read",
+				...,
```

Comment:
this line is invalid JSON, which is breaking lexing and therefore syntax highlighting

Reply:
If you're concerned that the list takes up too much space, you could always create a new Role with only one or two Permissions and then show that request/response - or don't, it's fine how it is IMO.

Reply:
Yeah I just used the `read-only` role instead, because that has lesser permissions than the `admin` role.